.github/workflows/push.yaml

```diff
@@ -34,6 +34,16 @@ jobs:
         with:
           path: self
 
+      - name: Storm - unit tests
+        env:
+          HELM_UNITTESTS_VERSION: 0.3.5
+          HELM_UNITTESTS_PLUGIN: https://github.com/helm-unittest/helm-unittest
+        working-directory: self/storm
+        shell: bash
+        run: |
+          helm plugin install ${HELM_UNITTESTS_PLUGIN} --version ${HELM_UNITTESTS_VERSION} > /dev/null
+          helm unittest . -o storm/tests/results/test-output.xml -t junit
+
```

Comment:
I don't think this is the right place to run those unit tests. This change would mean that when Armada wants to push an updated chart, we will run the Storm chart unit tests. I think what we want instead is to have an opt-in property in `config.json` that tells us whether a chart supports unit tests, in which case we will run them.